src/ui/IframeDialog.tsx

```diff
@@ -22,6 +24,7 @@ export const IframeDialog = forwardRef<HTMLIFrameElement, IframeModalProps>(
     const [isDialogOpen, setIsDialogOpen] = useState(isOpen);
     const [dialogHeight, setDialogHeight] = useState('0px');
     const [isIframeLoaded, setIsIframeLoaded] = useState(false);
```

Comment:
I mentioned on the fast-auth-signer review, but just to clarify. I think it would be easier if we default the modal to hidden instead of notHidden.

Reply:
Also, I find more readable to name booleans with `is/has`. So it could be something like `isModalOpen`, so we know it's a boolean when we read and also know it's state

Reply:
I'm not sure about this. If we want to default to hidden and only show if ready approach, we actually need to make changes on more places. (Currently we have 3 scenario on keys and two of them are show by default)

Also current implementation expects iframe to trigger `{ hideModal: true }` which is already being used on general Sign . If we change this behaviour, it could also affect existing implementation. 

Let discuss more in depth about what would be benefit of default to hide and why it is important VS what this PR presents

Reply:
No worries Danny, I thought it would be a quick improvement. But if it may not be a improvement and it's already the codebase standard, don't worry about this. 

It's not a big deal for me.